File: node.js/clients/spot/examples/MarketData/Kline.js
/**
 * Public REST — Kline (Kline)
 * API docs: Spot v3 API → Market Data Endpoints
 * Run: node examples/MarketData/Kline.js
 */
const { createPublicClient, printResponse, printError } = require('../_lib')

const client = createPublicClient()

client.Kline({ symbol: 'BTCUSDT', interval: '1m', limit: '10' })
  .then(printResponse)
  .catch(printError)

File: go/clients/spot/examples/MarketData/Kline.go
package main

import (
	"fmt"

	"{GO_MOD}/clients/spot"
	spotv3 "{GO_MOD}/common/spotv3"
)

func main() {
	client := spot.NewClient(spotv3.Config{})
	api := client.RestApi()

	result := api.Kline(`{"symbol":"BTCUSDT","interval":"1m","limit":"10"}`)
	fmt.Println(result)
}
